Update trust anchor rotation instructions (#2080)
linkerd/website: linkerd.io/content/2-edge/tasks/automatically-rotating-control-plane-tls-credentials.md

    | base64 -d | inspect_cert
```

> Verify that the `linkerd-identity-trust-roots` ConfigMap should now contain

Reviewer: ```suggestion
Verify that the `linkerd-identity-trust-roots` ConfigMap now contains
```

the Subject keys from both Secrets.

```bash
kubectl get configmap -n linkerd linkerd-identity-trust-roots \
        -o jsonpath='{ .data.ca-bundle\.crt }' \
        | inspect_cert
```

Note that mTLS communication is still working fine at this point. All the
proxies in the data plane are still using the old identity issuer, signed by the
old trust anchor, and that trust anchor is still present in
`linkerd-identity-trust-roots`.

#### 2. Restart the control plane

Restart the control plane with `kubectl rollout restart`:

```bash
kubectl rollout restart -n linkerd deploy
kubectl rollout status -n linkerd deploy
```

This will cause the control plane to pick up the new trust anchor bundle.

#### 3. Restart the data plane

At this point, you'll need to restart your workloads as well, to force the
proxies to have the new trust anchor bundle. The exact mechanism to do this

> when depend on your workloads, but it's often just `kubectl rollout restart` for

Reviewer: ```suggestion
will depend on your workloads, but it's often just `kubectl rollout restart` for
```

🤦‍♂️

This will cause the control plane to pick up the new trust anchor and identity
issuer.

#### 6. Restart the data plane

Again, you'll need to restart your workloads, to force the
proxies to switch to the new identity issuer that uses the trust anchor bundle.

> The exact mechanism to do this when depend on your workloads, but it's often

Reviewer: ```suggestion
The exact mechanism to do this will depend on your workloads, but it's often
```

🤦‍♂️
